### ISSUE TITLE
create folder node_modules when build

### ISSUE THREAD

**Issue by fyzhu** [vuejs/vitepress#1294]
### Describe the bug

<img width="294" alt="image" src="https://user-images.githubusercontent.com/5175751/188640211-1fe92335-3fbd-4a01-9aa0-c7efd88caee9.png">

### Reproduction

npm run build

### Expected behavior

no unexpected folder

### System Info

```shell
System:
    OS: macOS 12.5.1
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
    Memory: 187.60 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.15.0 - ~/.nvm/versions/node/v16.15.0/bin/node
    Yarn: 3.0.2 - /usr/local/bin/yarn
    npm: 8.5.5 - ~/.nvm/versions/node/v16.15.0/bin/npm
  Browsers:
    Chrome: 104.0.5112.101
    Firefox: 88.0
    Safari: 15.6.1
  npmPackages:
    vitepress: 1.0.0-alpha.8 => 1.0.0-alpha.13
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by brc-dd** [comment]
> npm run build

on what? Please provide complete reproduction.

**Comment by fyzhu** [comment]
> > npm run build
> 
> on what? Please provide complete reproduction.

Any project base on vitepress has this problem

**Comment by fyzhu** [comment]
create folder node_modules in docs when linkVue
but don't delete folder node_modules when unlinkVue

**Comment by brc-dd** [comment]
fixed in #1337
